curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/657e2ecee9f60092c89338d9.md

```diff
@@ -1,28 +1,29 @@
 ---
 id: 657e2ecee9f60092c89338d9
-title: Step 84
+title: Step 85
 challengeType: 0
-dashedName: step-84
+dashedName: step-85
 ---
 
 # --description--
 
-If the user rolls a small straight, then the `Small straight` radio button option should be enabled with a score of `30` points.
+If the user rolls a small straight, then the `Small straight` radio button option should be enabled with a score of 30 points.
 
-Start by creating an `if` statement to check if `uniqueNumbersStr` is included in the `smallStraightsArr`. If so, call the `updateRadioOption` with `3` and `30` for the arguments. 
+Start by creating an `if` statement to check if any element in `smallStraightsArr` is included in `uniqueNumbersStr`. If so, call the `updateRadioOption` function with arguments `3` and `30`.
 
 # --hints--
 
 You should use the `includes` method to check if `uniqueNumbersStr` is included in the `smallStraightsArr`.
 
 ```js
-assert.match(code, /if\s*\(\s*smallStraightsArr\.includes\(\s*uniqueNumbersStr\s*\)\s*\)\s*{?/);
+assert.match(code, /if\s*\(\s*smallStraightsArr\.some\(\s*straight\s*=>\s*uniqueNumbersStr\.includes\(\s*straight\s*\)\s*\)\s*\)\s*{?/);
```

Comment:
The issue is that we are expecting the camper to arrive at this solution.
However the directions provide no indication that the `some` method needs to be used here or that the callback for the `some` method need a parameter of `straight`. 

This test has multiple correct answers.
If you look at the original [forum post](https://forum.freecodecamp.org/t/learn-intermediate-algorithmic-thinking-by-building-a-dice-game-step-87/679303/5), they originally came up with this code 

```js
smallStraightArr.forEach(straight => {
  if(uniqueNumberStr.includes(straight)) {
    updateRadioOption(3, 30);
  }
}
```

so we should look into ways to allow for multiple valid answers and write a more flexible test that don't involve a regex like this

Reply:
The `some` method is a better approach because the `updateRadioOption(3, 30);` function would only happen once. 

If we do want the campers to only arrive at this solution, then the directions need to be update to do a better job of leading them there.
Also the hints would need to better illustrate the solution.
Otherwise campers will struggle to get there